src/pint/observatory/topo_obs.py

```diff
@@ -177,6 +204,11 @@ def __init__(
         self.origin = origin
         super().__init__(name, aliases=aliases)
 
+    def __repr__(self):
+        aliases = [f"'{x}'" for x in self.aliases]
+        s = f"TopoObs '{self.name}' ({','.join(aliases)}) at [{self._loc_itrf.x}, {self._loc_itrf.y} {self._loc_itrf.z}]:\n{self.origin}"
```

Comment:
`repr` is usually supposed to at least look  like python code that might reproduce the object. So `TopoObs(...)` might be more conventional.